Sources/KeyboardShortcuts/Localization/zh-TW.lproj/Localizable.strings

```diff
@@ -0,0 +1,5 @@
+"record_shortcut" = "記錄快捷鍵";
+"press_shortcut" = "鍵入快捷鍵";
+"keyboard_shortcut_used_by_menu_item" = "此快捷鍵無法使用，因為它已被選單項目「%@」使用。";
+"keyboard_shortcut_used_by_system" = "此快捷鍵無法使用，因為它已是系統通用快捷鍵。";
+"keyboard_shortcuts_can_be_changed" = "可以在「系統偏好設置 › 鍵盤 › 快速鍵」中更改大多數的系統快捷鍵。";
```

Comment:
```suggestion
"keyboard_shortcuts_can_be_changed" = "可以在「系統偏好設定 › 鍵盤 › 快速鍵」中更改大多數的系統快捷鍵。";
```

Reply:
Thank you @hank121314 for spotting the typo. 
Appreciate it!